addons/mail/static/tests/tours/mail_composer_test_tour.js

```diff
@@ -264,3 +264,38 @@ registry.category("web_tour.tours").add("mail/static/tests/tours/mail_composer_t
         },
     ],
 });
+
+registry.category("web_tour.tours").add("mail_composer_attachment_removal_tour", {
+    steps: () => [
+
+    {
+        content: "click on send by email",
+        trigger: ".o_statusbar_buttons > button[name='action_quotation_send']",
+        run: "click"
+    },
+    {
+        content: "save a new layout",
+        trigger: ".o_technical_modal button[name='document_layout_save'], .o_field_widget[name='attachment_ids'] li > button",
+        run() {
+            document.querySelector(
+                ".o_technical_modal button[name='document_layout_save']"
+            )?.click();
+        }
+    },
```

Comment:
This could just be
```
content: "save a new layout",
trigger: ".o_technical_modal button[name='document_layout_save']",
run: "click"
```

The `li > button` part is implicitly taken into account in the next step anyway.